docs/api.md

```diff
@@ -1361,7 +1361,7 @@ most.from([1, 2, 3, 4])
 ### switch
 
 ####`stream.switch() -> Stream`
-####`most.switch(stream) -> Stream`
+####`most.switchLatest(stream) -> Stream`
```

Comment:
Sounds like the right direction to me :+1:

In terms of immediate action on this PR, I could see something like:

- Change the TOC to say "switchLatest, alias switch"
- Change `stream.switch()` to `stream.switchLatest` in the heading, examples, and body text
- Add "Alias: stream.switch, most.switch" just below the heading [like we've done in other places](https://github.com/Frikki/most/blob/bee24bb1dae1cad20ffabd8d8efa9e7036c5485f/docs/api.md#chain)

Then, we can take the additional action to deprecate all the aliases we plan to delete for 2.0.  Sound right?